defaults/config.js

```diff
@@ -236,10 +236,13 @@ module.exports = {
 		//
 		// Nick
 		//
+		// Percent sign (%) will be replaced into a random number from 0 to 9.
+		// For example, Guest%%% will become Guest123 on page load.
+		//
 		// @type     string
-		// @default  "lounge-user"
+		// @default  "thelounger%%"
 		//
-		nick: "lounge-user",
+		nick: "thelounger%%",
```

Comment:
I'd prefer to have `thelounge-user%%` but whatever.

Reply:
We can do `TheLounge%%`

Reply:
lol no thanks. `thelounge%%`?